### ISSUE TITLE
QP: solver does not terminate on a very small input file

### ISSUE THREAD

**Issue by dirkschumacher** [ERGO-Code/HiGHS#1000]
The following problem results in HiGHS not terminating (in reasonable time at least).

### Input
```
\ File written by HiGHS .lp file handler
min
 obj: + [ +0.0024 x1 * x1 +0.0048 x1 * x2 +0.0024 x1 * x3 +0.0044 x2 * x2 +0.0032 x2 * x3 +0.0032 x3 * x3  ]/2 
st
 con1: +1 x1 +1 x2 +1 x3 = +1
bounds
 0 <= x1 <= 1
 0 <= x2 <= 1
 0 <= x3 <= 1
end
```

### Observed behaviour:

```
> ./highs model.lp
Running HiGHS 1.2.1 [date: , git hash: ]
Copyright (c) 2022 ERGO-Code under MIT licence terms
WARNING: Ignored 3 entries of Hessian in opposite triangle
QP   model has 1 rows; 3 cols; 3 matrix nonzeros; 6 Hessian nonzeros
0, 0.001600, 0, 0.000702, 0.000000, 0, 0.000000, 0.000000
1000, 0.001600, 0, 0.002524, 0.000000, 0, 0.000000, 0.000000
...
562000, 0.001600, 0, 0.743409, 0.000000, 0, 0.000000, 0.000000
563000, 0.001600, 0, 0.744670, 0.000000, 0, 0.000000, 0.000000
564000, 0.001600, 0, 0.745968, 0.000000, 0, 0.000000, 0.000000
565000, 0.001600, 0, 0.747275, 0.000000, 0, 0.000000, 0.000000
566000, 0.001600, 0, 0.748554, 0.000000, 0, 0.000000, 0.000000
567000, 0.001600, 0, 0.749856, 0.000000, 0, 0.000000, 0.000000
...
21061000, 0.001600, 0, 27.539831, 0.000000, 0, 0.000000, 0.000000
21062000, 0.001600, 0, 27.541118, 0.000000, 0, 0.000000, 0.000000
...
```

### Additional information

Removing the `con1` constraint makes the solver terminate

```
Running HiGHS 1.2.1 [date: , git hash: ]
Copyright (c) 2022 ERGO-Code under MIT licence terms
WARNING: Ignored 3 entries of Hessian in opposite triangle
QP   model2 has 0 rows; 3 cols; 0 matrix nonzeros; 6 Hessian nonzeros
0, 0.000000, 0, 0.000445, 0.000000, 0, 0.000000, 0.000000
1, 0.000000, 0, 0.000456, 0.000000, 0, 0.000000, 0.000000
Model   status      : Optimal
QP ASM    iterations: 1
Objective value     :  0.0000000000e+00
HiGHS run time      :          0.00
```

System: darwin/arm64 (M1), precompiled static binary of HiGHS.

**Comment by jajhall** [comment]
Very odd. If the objective coefficients are increased by a factor of 10, then it still fails. However, after another factor of 10 increase the problem is solved easily.

**Comment by jajhall** [comment]
[issue-1000.lp.txt](https://github.com/ERGO-Code/HiGHS/files/10450893/issue-1000.lp.txt)

**Comment by feldmeier** [comment]
Should be fixed by #1282
